.circleci/config.yml

```diff
@@ -218,12 +202,57 @@ jobs:
       - prepare_docker
       - prepare_docker_buildx
       - run:
-          name: Set environment
+          name: Set environment settings
+          command: |
+            export DOCKER_IMAGE_SIGN="<< parameters.docker-image-sign >>"
+            export DOCKER_IMAGE_PLATFORMS="<< parameters.docker-image-platforms >>"
+            export DOCKER_IMAGE_VERSION="<< parameters.docker-distroless-version >>"
+            export DOCKER_IMAGE_REF="<< parameters.docker-image-registry >>/<< parameters.docker-image-tag >>"
+      - run:
+          name: Set environment settings
+          no_output_timeout: 60m
           command: |
-            echo 'export DOCKER_IMAGE_PLATFORMS="<< parameters.docker-image-platforms >>"' >> $BASH_ENV
-            echo 'export DOCKER_IMAGE_VERSION="<< parameters.docker-distroless-version >>"' >> $BASH_ENV
-            echo "export DOCKER_IMAGE_TAG=\"<< parameters.docker-image-registry >>/<< parameters.docker-image-tag >>\"" >> $BASH_ENV
-      - docker_multiarch_build
+            for PLATFORM in $DOCKER_IMAGE_PLATFORMS; do
+              TARGETOS="$(echo "${PLATFORM}" | cut -d '/' -f 1)"
+              TARGETARCH="$(echo "${PLATFORM}" | cut -d '/' -f 2)"
+
+              if [ $TARGETOS != "linux" ]; then
+                echo "Targeted OS not supported $TARGETOS"
+                exit 1
+              fi
+
+              if [ $TARGETARCH == "amd64" ]; then
+                DOCKER_IMAGE_TAG="${DOCKER_IMAGE_REF}"
+              else
+                DOCKER_IMAGE_TAG="${DOCKER_IMAGE_REF}.${TARGETARCH}"
+              fi
```

Comment:
We are loosing the `amd64` tag of the image, maybe we should introduce a new convention here with explicitly tagged arch ?

Reply:
I decided to follow similar strategy than `mysql-server` where the amd64 is consider the default and the arm64 is tagged specifying the arch, although I did a different naming convention which I would be open to discuss if you propose something else

![image](https://user-images.githubusercontent.com/2728080/147915289-a5512e70-5388-4a51-9473-238d4bd62564.png)

https://hub.docker.com/r/mysql/mysql-server/tags?page=1